deploy/sample-backup-configs/sample_config_build_recipes.yaml

```diff
@@ -0,0 +1,112 @@
+# Build-time design configuration for the FireSim Simulation Manager
+# See docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst for documentation of all of these params.
+
+# this file contains sections that describe hardware designs that /can/ be built.
+# edit config_build.yaml to actually "turn on" a config to be built when you run
+# buildafi
+
+# Example Build Recipe
+# example-recipe-name:
+#    # DESIGN SPECIFIC
+#    DESIGN: TopModule
+#    TARGET_CONFIG: Config
+#    deploy-triplet: null
+#    # RUN PLATFORM SPECIFIC
+#    PLATFORM_CONFIG: Config
+#    post-build-hook: null
+#    s3-bucket-name: firesim-AWSUSERNAME
+#    # BUILD PLATFORM SPECIFIC
+#    build-farm: default-build-farm
+
+# Quad-core, Rocket-based recipes
+firesim-rocket-quadcore-nic-l2-llc4mb-ddr3:
+    DESIGN: FireSim
+    TARGET_CONFIG: WithNIC_DDR3FRFCFSLLC4MB_WithDefaultFireSimBridges_WithFireSimHighPerfConfigTweaks_chipyard.QuadRocketConfig
+    deploy-triplet: null
+    PLATFORM_CONFIG: WithAutoILA_F90MHz_BaseF1Config
+    post-build-hook: null
+    s3-bucket-name: firesim-AWSUSERNAME
+    build-farm: default-build-farm
```

Comment:
Throughout the `sample_` yamls I overspecify what `build-farm` (and other keys) are needed. In reality you can remove this key and it would default to `default-build-farm` (this is documented in the `sample_config_build_farm.yaml` file at the top but will need to be further documented in the readthedocs). As for the `s3-bucket-name`, this is a bit trickier. Ideally in the next PR ("what-to-build" #853), this key is associated with how the bitstream is build (since the bucket name is only used in AFI create IIRC). So in this PR I just moved it here in the meantime.

Reply:
Ok.  I'll keep this in mind as I read through #853